README.md

```diff
@@ -37,7 +37,7 @@ cp .env.sample .env
 docker-compose up -d
 ```
 
-A [Jupyter Notebook](http://localhost:8888/tree/feast/examples) is now available to start using Feast.
+A [Jupyter Notebook](https://github.com/gojek/feast/blob/master/examples/basic/basic.ipynb) is now available to start using Feast.
```

Comment:
This link should be available once you spin up Feast using Docker Compose. It shouldnt directly link to the source repository.

Reply:
Did you have trouble accessing the notebook?

Reply:
It was totally my mistake. After reading some information of the project (Medium, web...) I reached the repo and "Jupyter Notebook" catched my attention at first time, so I clicked and it didn't work. Now that you explained it and I carefully read the whole readme, I understand why: it's not meant to be as an standalone notebook to show feast way of work, but as a way to test the docker compose and start working from there.
My apologies.

Reply:
Maybe it should have been made clearer? I think it might catch some folks off guard.

Reply:
The docker compose section I think is well explained. But maybe something on the line of:

> A basic (yet complete) example of using feast can be found in this [Jupyter Notebook](https://github.com/gojek/feast/blob/master/examples/basic/basic.ipynb).

just below this line

> prediction = my_model.predict(fs.get_online_features(customer_features, customer_entities))

could help to avoid these mistakes. What do you think?